compiler/src/iree/compiler/Codegen/Utils/Utils.cpp

```diff
@@ -799,6 +799,37 @@ FailureOr<int64_t> getSoftwarePipelineStoreStage(DictionaryAttr config) {
   return llvm::cast<IntegerAttr>(stage).getInt();
 }
 
+int getReductionTilingFactor(int64_t dimSize) {
+  if (dimSize % 4 == 0)
```

Comment:
Might be worth putting some comments mentioned in https://github.com/iree-org/iree/pull/12922 here about the preference for 4 either here or at the new call site.

Reply:
I imported the comment that was there previously, and there is already a longer comment near the call site too.